common/algorithms/hashes.h

```diff
@@ -47,9 +50,8 @@ size_t hash_range(const Rng &range, Hasher hasher = Hasher()) {
 template<class Rng, class Hasher = std::hash<range_value_type<Rng>>>
 class range_hasher : Hasher {
 public:
-  explicit range_hasher(Hasher hasher = Hasher()) :
-    Hasher(std::move(hasher)) {
-  }
+  explicit range_hasher(Hasher hasher = Hasher())
+    : Hasher(std::move(hasher)) {}
```

Comment:
Extra diff here and in other places in this file

Reply:
Fixed